cdap-ui/app/cdap/components/KeyValuePairs/index.js

```diff
@@ -108,9 +110,17 @@ export default class KeyValuePairs extends Component {
           return (
             <div key={pair.uniqueId}>
               <Provider store={this.keyValueStore}>
-                <KeyValuePairCopy
-                  index={index}
-                />
+                {
+                  this.props.getResettedKeyValue ?
+                    (
+                      <KeyValuePairCopy
+                        index={index}
+                        getResettedKeyValue={this.props.getResettedKeyValue.bind(this, index)}
```

Comment:
Not sure if we should we bind `this` to a function `KeyValuePairs` component gets as props.